File: scala/play2/json_post_sample/app/controllers/Application.scala
package controllers

import play.api._
import play.api.mvc._
import play.api.libs.json._

object Application extends Controller {

	def index = Action {
		Ok(views.html.index())
	}

	def send = Action { req =>
		println("--- query string ---")
		println(req.queryString)
		
		println("--- json ---")
		println(req.body.asJson)
		
		Ok("")
	}

}

File: java/play2/json_sample/app/controllers/Application.java
package controllers;

import play.*;
import play.mvc.*;

import views.html.*;

import org.codehaus.jackson.JsonNode;

public class Application extends Controller {

	public static Result index() {
		return ok(index.render());
	}

	@BodyParser.Of(BodyParser.Json.class)
	public static Result send() {
		System.out.println("--- queryString ---");
		System.out.println(request().queryString());

		System.out.println("--- body ---");
		JsonNode json = request().body().asJson();
		System.out.println(json);

		return ok();
	}
  
}